lib/services/registered_addons.pm

```diff
@@ -78,10 +79,23 @@ sub check_suseconnect {
     diag "@addons";
 }
 
+sub check_suseconnect_cmd {
+    my $ls_out = script_output("SUSEConnect --list-extensions", 120);
+    diag "$ls_out";
+    my $status_out = script_output("SUSEConnect --status-text", 120);
+    diag "$status_out";
+    for (my $i = 1; $i < @addons; $i = $i + 1) {
+        diag "$addons[$i]";
+        die "$addons[$i] is not existed at SUSEConnect --list-extensions" if ($ls_out     !~ /Deactivate(.*)$addons[$i]/);
```

Comment:
> BTW: i think we need enable this check not only in service check, but also need enable for migration group, since this is very important part for migration automation check

At system_prepare, we have checked SUSEConnect --status-text. But at this part, we can check each modules, as we have the checked code before.